formatter/src/formatter/mac.rs

```diff
@@ -85,7 +91,7 @@ impl Formatter<'_> {
             .cbox((parent_indent.tabs * self.settings.tab_spaces + parent_indent.spaces) as isize);
 
         self.flush_comments(cx.span().start().line - 1);
-        let macro_word = format!("{}! {{", self.settings.html_macro);
+        let macro_word = format!("{}! {{", view_mac.format_macro);
```

Comment:
I wonder if we have to pass `format_macro` here as we already have access to the macro that syn parsed.
Can we use something like `view_mac.mac.path` instead?

Reply:
Yeah that makes much more sense